src/ra_rules.ruleset

```diff
@@ -3,9 +3,10 @@
   <Include Path="cppcorecheckclassrules.ruleset" Action="Default" />
   <Include Path="cppcorecheckconstrules.ruleset" Action="Default" />
   <Include Path="cppcorecheckfuncrules.ruleset" Action="Default" />
+  <Include Path="cppcorecheckgslrules.ruleset" Action="Default" />
```

Comment:
is this file even still used? it looks like you switched to `CppCoreCheckGslRules.ruleset`